lib/service/connect.go

```diff
@@ -1126,14 +1128,81 @@ func (process *TeleportProcess) getConnector(clientIdentity, serverIdentity *sta
 	}
 
 	// Set cluster features and return successfully with a working connector.
-	process.setClusterFeatures(pingResponse.GetServerFeatures())
+	// todo (michellescripts) remove clone & compatibility check in v18
```

Comment:
Should this be removed in v18 or v17? I'm guessing we'll need to backport this to v16, then once we release v17 off of current master we'll expect the entitlements to have already been set appropriately?

In that case, we probably don't even need master PR and only the branch/v16 backport?

Reply:
You're technically correct, but I have a ton of backwards work marked for removal in v18 and I think it will be easier to keep the branches consistent

Reply:
Happy to open straight to v16 if you have a strong preference

Reply:
No strong preference, up to you.

Reply:
We need it at least till 17